app/templates/application.hbs

```diff
@@ -1,5 +1,17 @@
-{{! The following component displays Ember's default welcome message. }}
-<WelcomePage />
-{{! Feel free to remove this! }}
-
-{{outlet}}
\ No newline at end of file
+<div class="bg-blue-100 h-screen w-screen">
+  <div class="absolute top-0 left-0">
+    <Battleship />
+    <Battleship class="small" />
+  </div>
+  <header class="conatiner mx-auto h-auto pb-4 sm:pb-10 pr-6 bg-blue-300 flex flex-wrap items-end sm:items-center">
```

Comment:
throw this in a component?

Reply:
Yeah, probably could be extracted, was just trying to build up the nav and got carried away.

Reply:
I think I'm actually going to rip out all the styling from this MR and make it only implementing tailwind. The rest is noise TBH